addons/custom-block-shape/addon.json

```diff
@@ -0,0 +1,112 @@
+{
+  "name": "Custom block shape",
+  "description": "Adjust the padding, corner radius, and notch height of Scratch blocks.",
+  "tags": ["editor", "theme", "beta"],
+  "credits": [
+    {
+      "name": "SheepTester",
+      "link": "https://scratch.mit.edu/users/Sheep_maker/"
+    },
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://scratch.mit.edu/users/lisa_wolfgang"
+    },
+    {
+      "name": "GarboMuffin",
+      "link": "https://scratch.mit.edu/users/GarboMuffin"
+    }
+  ],
+  "enabledByDefault": false,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "info": [
+    {
+      "type": "warning",
+      "text": "Decreasing the padding size is only visible to you, so when your projects are viewed by other users, your scripts may overlap.",
+      "id": "paddingWarning"
+    }
+  ],
+  "settings": [
+    {
+      "name": "Padding size (50-200%)",
+      "id": "paddingSize",
+      "type": "integer",
+      "min": 50,
+      "default": 100,
+      "max": 200
+    },
+    {
+      "name": "Corner size (0-300%)",
+      "id": "cornerSize",
+      "type": "integer",
+      "min": 0,
+      "default": 100,
+      "max": 300
+    },
+    {
+      "name": "Notch height (0-150%)",
```

Comment:
The text in parentheses isn't needed I don't think since the element has min and max attrs. But if you decide to keep it, I don't think there should be a percent there

Reply:
I think it's helpful to know the possible parameters that can be entered, and the percent helps people visualize the effect that the setting will have.

Reply:
I agree with @lisa-wolfgang... users aren't expected to inspect the page to figure that out.